### DISCUSSION TITLE
HAM Radio Frequency

### DISCUSSION THREAD

**Discussion by RobFather1** [StateFarm-STEM/hablogger#146] [Q&A]
Do we know what frequency the balloons will be transmitting on?  I ask because my dad is a HAM and would like to try to monitor the balloon launch via his radio. Thanks Rob

**Comment by pankonen** [comment]
Yes! They will all be on APRS on VHF (144.39) and some will be using WSPR on HF. We usually pick the WSPR frequency when we configure the trackers just before flight based on band conditions. We will also be using a hand full of callsigns for the various balloons we are flying this year (8).  You can follow them on the radio and on website aprs.fi and searching for the balloon callsign. We are working on some LoRa and pictures and videos on 70cm but its been slow going. iw ill try and update this issue after we get trackers configured with what to have him look for. Thanks, Tom KD9SAT
